### ISSUE TITLE
Webhooks plugin: deprecated TypeORM findOne method signatures used in v7.0.0

### ISSUE THREAD

**Issue by yasserlens** [Pinelab-studio/pinelab-vendure-plugins#189]
**Which plugin/repository is the issue about?**
vendure-plugin-webhook@7.0.0

**Describe the bug**
It appears that the plugin uses deprecated findOne signatures. 
This is what I see under node_modules/.../vendure-plugin-webhook/dist/api/webhook.service.js

```
 async getWebhook(channelId) {
        return this.connection
            .getRepository(webhook_per_channel_entity_1.WebhookPerChannelEntity)
            .findOne({ channelId });
    }
```
but it should be 
```
 async getWebhook(channelId) {
        return this.connection
            .getRepository(webhook_per_channel_entity_1.WebhookPerChannelEntity)
            .findOne({ where: { id: channelId } });
    }
```
The same findOne signature is under "saveWebhook"

Having said that - getWebhook and saveWebhook are no longer there in the latest version on github. 
The code I see in node_modules does not include the following commit on 17 May 2023:
https://github.com/Pinelab-studio/pinelab-vendure-plugins/commit/ec54ccf35a27b34075eb60dbca88a75d7cc9f6c1

**To Reproduce**
- npm i vendure-plugin-webhook
- add the webhook plugin to vendure.config.ts (both the server plugin and admin-ui plugin)
- npm run dev
- go to the webhooks page under "Settings"
- A graphQL error will return from the server
```
{"errors":[{"message":"You must provide selection conditions in order to find a single row.","locations":[{"line":2,"column":3}],"path":["updateWebhook"]}],"data":{"updateWebhook":null}}
```
- This is due to the findOne({channelId}) method mentioned above
- To double check the version I ran
```
npm list vendure-plugin-webhook
```
And got
```
stores@0.1.0 /.../vendure/backend
└── vendure-plugin-webhook@7.0.0
```

**Expected behavior**
The correct vendure-plugin-webhook version to be installed (the latest) which matches the published version 7.0.0 - instead of an older version which has deprecated TypeOrm methods like findOne's old signature.

**Screenshots**

<img width="559" alt="Screenshot 2023-07-03 at 10 18 33 a m" src="https://github.com/Pinelab-studio/pinelab-vendure-plugins/assets/2945770/c12082ab-1780-477b-b8d1-73418debf2f4">
<img width="581" alt="Screenshot 2023-07-03 at 10 18 48 a m" src="https://github.com/Pinelab-studio/pinelab-vendure-plugins/assets/2945770/664774e3-1b40-4ef2-97c6-cae00323989f">

**Environment**

- NodeJS version: irrelevant
- Vendure version: 2.0.0
- Plugin version 7.0.0
- Database PostgreSQL

**Comment by yasserlens** [comment]
Turns out v7.0.0 has the issues described above - they're fixed in v7.0.6 but that's not published yet.
https://github.com/Pinelab-studio/pinelab-vendure-plugins/releases/tag/v7.0.6

Could you please publish it?

**Comment by martijnvdbrug** [comment]
Will do, thanks for the report!

**Comment by yasserlens** [comment]
Hey Martijn - do you think you'll get to work on this soon? If you could let me know so I know whether to branch out or not.
Many thanks!

**Comment by martijnvdbrug** [comment]
@yasserlens It's published as 7.1.1. now!

(It was actually already published, but with a v2 tag instead of `latest`. Versions are a bit messed up because we are in the middle of upgrading all plugins to V2)
